platform/posix/transport/include/openssl_posix.h

```diff
@@ -120,7 +120,7 @@ typedef struct OpensslCredentials
      * @note By setting this to 0, OpenSSL uses the default value,
      * which is 16384 (SSL3_RT_MAX_PLAIN_LENGTH).
      */
-    size_t maxFragmentLength;
+    uint64_t maxFragmentLength;
```

Comment:
Why is this type change needed?

Reply:
In most of the openssl functions in which maxFragmentLength is used, an explicit cast was required to uint64_t. Instead of explicitly casting it everywhere, changed the type to uint64_t.

Reply:
Okay, I asked primarily in relation to the `alpnProtosLen` member which uses a `size_t` type. Should it be fine to change that member's type as well for consistency?

Reply:
Yea. That is a good point. I will change the type of that as well. Thanks.